apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/DocumentEditor.tsx

```diff
@@ -65,34 +67,14 @@ const HISTORY_ELM_HEIGHT = 44
  * @param props.initialDiff - Initial diff data for the document
  */
 function DocumentEditorContent({
-  document: _document,
-  documents: _documents,
   freeRunsCount,
   initialDiff,
 }: DocumentEditorProps) {
-  const { updateDocumentContent } = useDocumentValue()
+  const { updateDocumentContent, document } = useDocumentValue()
```

Comment:
better not to export document from this context, the document source of truth is the document store exporting it from other places adds duplicity which leads to confusion